example/example.ipynb

```diff
@@ -39,20 +39,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 15,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "174it [00:01, 103.35it/s, tot=174, good=174, rec=2124, err=0]\n",
-      "193it [00:01, 100.70it/s, tot=193, good=193, rec=2553, err=0]\n",
-      "194it [00:01, 98.60it/s, tot=194, good=194, rec=2596, err=0]\n",
-      "219it [00:02, 101.11it/s, tot=219, good=219, rec=2914, err=0]\n"
-     ]
-    }
-   ],
+   "outputs": [],
```

Comment:
The table generation progress log is now missing. Not sure why. Perhaps a restart and run all would fix it?